tests/de/gurkenlabs/litiengine/gui/NumberAdjusterTests.java

```diff
@@ -0,0 +1,53 @@
+package de.gurkenlabs.litiengine.gui;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.input.Input;
+
+public class NumberAdjusterTests {
```

Comment:
These changes are also contained in [PR #383](https://github.com/gurkenlabs/litiengine/pull/383/files#diff-3ab447b9083364a5b86a96eb4dd5e406701b43b08b520e519d99a32e203ce64a). Let's hope there won't be any merge conflicts :)

Reply:
Mistake from my side. I just kept working with the same file by adding new tests.